components/Starknet/modules/starknet_versions/pages/version_notes.adoc

```diff
@@ -16,8 +16,26 @@ include::ROOT:partial$snippet_important_goerli2_removed.adoc[]
 |Environment |Starknet version|Sierra version|Cairo version
 
 |Mainnet|0.13.1|1.5.0|2.0.0 - 2.6
-|Sepolia Testnet|0.13.1|1.5.0|2.0.0 - 2.6
-|Goerli Testnet (deprecated)|0.13.1|1.5.0|2.0.0 - 2.6
+|Sepolia Testnet|0.13.1.1|1.5.0|2.0.0 - 2.6
+|===
+
+[id="version0.13.1.1"]
+== Starknet v0.13.1.1
+
```

Comment:
I think that we should add a sentence in the spirit of the mailing list update - following community feedback regarding high declaration prices ...

Reply:
@ArielElp PTAL.